textproc/lilypond/Portfile

```diff
@@ -206,14 +229,29 @@ post-patch {
         scripts/auxiliar/translations-status.py \
         scripts/auxiliar/update-snippets.py
 
+    # ImageMagick and xelatex (or pdflatex) are only needed for
+    # complete documentation builds, which +docs doesn't do currently.
+    # Ditto for Cyrillic stuff (i.e., testing for 'fikparm.mf').
+    if {[variant_isset docs]} {
+        reinplace -W ${worksrcpath} \
+            "s|\\(IMAGEMAGICK.*\\), \\\$DOCUMENTATION_REQUIRED|\\1, OPTIONAL|g" \
```

Comment:
Since you don't want variable interpolation in there strings, you could avoid a lot of backslashes by quoting the strings using braces `{` `}` instead of double quotes `"` `"`. Even better, if the reinplaces don't change very many lines, use a patch file instead of reinplaces.

Reply:
Thanks for the tip.  Fixed.

I must admit that I prefer reinplaces over patch files here; it's easier to handle if everything is in a single file.

Reply:
Please use patchfiles instead of reinplaces in the future, unless you need to use a variable from Tcl or unless the change will affect a huge number of lines. Patchfiles provide context that can help the next person who updates the port.